src/test/java/org/wiremock/integrations/testcontainers/WireMockContainerBannerTest.java

```diff
@@ -21,7 +21,7 @@
 
 class WireMockContainerBannerTest {
 
-    WireMockContainer wireMockContainer = new WireMockContainer("2.35.0");
+    WireMockContainer wireMockContainer = new WireMockContainer(WireMockContainer.WIREMOCK_2_LATEST);
```

Comment:
Maybe wold be better to use constructor without params

Reply:
I will later create a TestConfig class, similar to https://github.com/wiremock/wiremock-docker/pull/67